binding/format/protobuf/v2/protobuf.go

```diff
@@ -76,7 +76,7 @@ func (protobufFmt) Unmarshal(b []byte, e *event.Event) error {
 }
 
 // convert an SDK event to a protobuf variant of the event that can be marshaled.
-func sdkToProto(e *event.Event) (*pb.CloudEvent, error) {
```

Comment:
The method is used but not renamed, and I am not sure this is a great name for an exported method. 

I would think `ToProto` and `FromProto` would be better names to export.